stock_inventory_turnover_report/models/product_product.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Open Source Intgerators
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from dateutil.relativedelta import relativedelta
+from odoo import fields, models
+
+
+class Product(models.Model):
+    _inherit = "product.product"
+
+    def _compute_quantities_in_dict(self, from_date):
```

Comment:
qty_available already considers locations in the context (gotten from the search view). This allows to analyze data by location, warehouse, company, etc..

 https://github.com/odoo/odoo/blob/7647077e9762359060b408fa923773826998fedf/addons/stock/models/product.py#L153

It'd be good to implement the same in this method. If you do this then this report could be filtered by location or warehouse :thinking: otherwise it's probably giving wrong data if any of these filters are active